projects/1.16/assets/evilcraft/evilcraft/lang/zh_cn.json

```
@@ -1 +1,765 @@
-{}
\ No newline at end of file
+{
+
+  "itemGroup.evilcraft": "邪恶工艺",
+  "general.evilcraft.info.empty": "空",
+  "general.evilcraft.description": " Minecraft的邪恶模组.",
+
+
+  "subtitles.evilcraft.effect.vengeancebeam.base": "复仇光束",
+  "subtitles.evilcraft.effect.vengeancebeam.start": "复仇光束开始",
+  "subtitles.evilcraft.effect.vengeancebeam.stop": "复仇光束结束",
+  "subtitles.evilcraft.mob.vengeancespirit.ambient": "复仇之魂低吼",
+  "subtitles.evilcraft.mob.vengeancespirit.death": "复仇之魂死亡",
+  "subtitles.evilcraft.effect.box.beam": "永恒的灵魂箱光束",
+  "subtitles.evilcraft.effect.page.flipsingle": "翻页",
+  "subtitles.evilcraft.effect.page.flipmultiple": "翻多页",
+
+
+  "block.evilcraft.inner_block.info": "方块: %s",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "block.evilcraft.inner_block.info": "方块：%s",
```
请将英文标点全部替换为中文标点。